### ISSUE TITLE
Allow to disable HostKey verification for iOS

### ISSUE THREAD

**Issue by alex-apriamashvili** [MarathonLabs/marathon#264]
### Problem
Currently, iOS vendor is using `SSHj` library to connect to a remote machine. To establish such connection a public ssh key is used. It seems that `SSHj` might have an issue with the host key verification as it is using the first match found among the server and local key-encryption algorithms. That means that In case if that match is not equal to an algorithm specified in the `known_hosts` file, the key-validation will fail.
### Solution
This issue might be solved by manually putting an entry of a type required by `SSHj` to the `known_hosts` file.
The list of possible keys might be obtained with the following command:
```ssh
ssh-keyscan -H <your-host-name>
```
The output of that command of a required type needs to be added to the `known_hosts` file manually.
#### Proposed change:
I believe that in some cases this key verification is not needed in the first place, so it might make sense to add a possibility to disable such verification by introducing a vendor configuration value. By default this verification needs to be enabled and disabling shall be done on demand.
```yaml
vendorConfiguration:
  disableHostKeyVerifier: true
  ...
```

**Comment by Malinskiy** [comment]
We need to add some information about this in the docs

The options is not implemented in #265 which will be merged with #238

**Comment by Malinskiy** [comment]
@alex-apriamashvili  I've provided a run configuration that takes care of this setup in dev experience in #269. This should help to get everyone into the development loop much faster.

**Comment by Malinskiy** [comment]
Closed by #726. The host key verification is not needed now for local runs. For ssh ones there is an option to supply OpenSSH-formatted known hosts or just ignore this check altogether
